clustering/faces/mojarra/src/main/java/org/wildfly/clustering/faces/mojarra/context/flash/ContextFlashSerializationContextInitializer.java

```diff
@@ -16,6 +16,8 @@
 import org.wildfly.clustering.marshalling.protostream.SerializationContextInitializer;
 import org.wildfly.security.manager.WildFlySecurityManager;
 
+import com.sun.faces.context.flash.ELFlash;
```

Comment:
nitpick only: this should just be along the other imports.

Reply:
Fixed